lib/src/source_visitor.dart

```diff
@@ -171,22 +171,17 @@ class SourceVisitor extends ThrowingAstVisitor {
     builder.endSpan();
   }
 
-  // TODO(rnystrom): Type annotate once analyzer publishes a version with the
-  // new AST type.
-  // TODO(rnystrom): Test.
-  visitAssertInitializer(node) {
-    _simpleStatement(node, () {
```

Comment:
Adding a trailing semicolon, mainly. It was a bug, uncaught because there were no tests for it since at the time the code was written, there wasn't a published version of analyzer that supported parsing the syntax.